packages/modules/viewer-react/src/types.ts

```diff
@@ -131,21 +141,53 @@ export interface ViewerInitializerParams extends ViewerIModelAppOptions {
   /** array of iTwin.js Extensions */
   extensions?: ExtensionProvider[];
 }
-
-export interface ConnectedViewerProps {
-  iTwinId: string;
+export type RequiredViewerProps = XOR<
+  XOR<ConnectedViewerProps, FileViewerProps>,
+  BlankViewerProps
+>;
+export type ViewerProps = RequiredViewerProps & ViewerCommonProps;
+export type ViewerLoaderProps = RequiredViewerProps & LoaderProps;
+export type ConnectedViewerProps = {
+  iTwinId: GuidString;
   iModelId: string;
```

Comment:
i know GuidString is just an alias for string, but i guess might as well change iModelId as well if we're changing iTwinId

Reply:
Yeah... I think I'll go back to string.